client/src/pages/SignUp/SignUp.tsx

```diff
@@ -39,25 +39,18 @@ export default function Register(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
-        <Box
-          display="flex"
-          justifyContent="space-between"
-          alignItems="flex-start"
-          flexDirection="column"
-          className={classes.authWrapper}
-        >
-          <AuthHeader linkTo="/login" asideText="Already have an account?" btnText="Login" />
+        <Box display="flex" alignItems="center" flexDirection="column" className={classes.authWrapper}>
           <Box width="100%" maxWidth={450} p={3} alignSelf="center">
             <Grid container>
               <Grid item xs>
                 <Typography className={classes.welcome} component="h1" variant="h5">
-                  Create an account
+                  <strong>Sign up</strong>
                 </Typography>
               </Grid>
             </Grid>
             <SignUpForm handleSubmit={handleSubmit} />
           </Box>
-          <Box p={1} alignSelf="center" />
+          <AuthHeader linkTo="/login" asideText="Already a member?" linkText="Login" />
```

Comment:
Can we rename this component to AuthFooter as we are using it to the bottom of the form?